src/kudu/consensus/consensus.proto

```diff
@@ -401,6 +401,15 @@ message VoteRequestPB {
   optional CandidateContext candidate_context = 8;
 }
 
+// Additional context that a voter sends back in the response to RequestVote()
+// rpc
+message VoterContext {
+  // Candidate was removed from the voter's committed config and is currently
```

Comment:
I think inlining this might not be a bad idea.

Reply:
Will retain it this way. Can be used in future to exchange additional info (like voter's apply lag etc) so that the candidate (on winning election) can make a more informed decision and decide to transfer leadership. New feature not yet implemented. Is that okay?